tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectTest.java

```diff
@@ -46,11 +46,13 @@ public class ObjectTest {
 
     private CompileResult checkInInitializerResult;
     private CompileResult checkFunctionReferencesResult;
+    private CompileResult objectCtrResult;
 
     @BeforeClass
     public void setUp() {
         checkInInitializerResult = BCompileUtil.compile("test-src/object/object_field_initializer_with_check.bal");
         checkFunctionReferencesResult = BCompileUtil.compile("test-src/object/object_function_pointer.bal");
+        objectCtrResult = BCompileUtil.compile("test-src/object/object_constructor.bal");
```

Comment:
Why are we adding this to `ObjectTest` and not `ObjectConstructorTest`?

Reply:
Changed